Ladybird/AppKit/UI/LadybirdWebView.h

```diff
@@ -86,6 +86,8 @@
 
 - (void)debugRequest:(ByteString const&)request argument:(ByteString const&)argument;
 
+- (void)setAutoplay:(BOOL)enabled;
```

Comment:
Let's call this `setEnableAutoplay` (here and in the files below) in AppKit to match Qt

Reply:
Doesn't look like there's a checkbox added to enable this in AppKit?